fastlane_bot/config/provider.py

```diff
@@ -190,12 +183,9 @@ def __init__(self, network: ConfigNetwork, **kwargs):
             address=self.w3.to_checksum_address(N.FASTLANE_CONTRACT_ADDRESS),
             abi=FAST_LANE_CONTRACT_ABI,
         )
-        self.ARB_CONTRACT_VERSION = self.BANCOR_ARBITRAGE_CONTRACT.caller.version()
 
-        reward_percent, max_profit = self.BANCOR_ARBITRAGE_CONTRACT.caller.rewards()
-
-        self.ARB_REWARD_PERCENTAGE = str(int(reward_percent) / 1000000)
-        self.ARB_MAX_PROFIT = str(int(max_profit) / (10**18))
+        self.ARB_CONTRACT_VERSION = self.BANCOR_ARBITRAGE_CONTRACT.caller.version()
+        self.ARB_REWARDS_PPM = self.BANCOR_ARBITRAGE_CONTRACT.caller.rewards()[0]
```

Comment:
I suppose you've done this alright, but it looks from the old code that self.BANCOR_ARBITRAGE_CONTRACT.caller.rewards()[0] was a string yet for you it is an int?

Reply:
Yes, no point in converting it to a string when it is later used for arithmetic computation.
You'd also note that the name 'percentage' was wrong, since it is a normalized value (between 0 and 1).
The correct naming would be 'portion', but I went further ahead and preserved the original (on-chain) resolution.
So it is now a PPM integer value, and the division by 1 million takes place during the actual computations.